ang/fe-creditnote.ang.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
is it possible to move these 3 files (fe-creditnote.ang.php, fe-creditnote.css and fe-creditnote.js) also under `fe-creditnote` ? or will things not work like that ?

Reply:
No, it's not possible. but I've moved the fe-creditnote.css  file into a separate folder

Reply:
Behold the doc https://docs.civicrm.org/dev/en/latest/framework/angular/files/